src/container/srv_container.c

```diff
@@ -1912,15 +1917,30 @@ cont_agg_eph_store(struct cont_svc *svc, uuid_t cont_uuid, uint64_t ec_agg_eph)
 		D_GOTO(out_lock, rc);
 	}
 
-	d_iov_set(&value, &ec_agg_eph, sizeof(ec_agg_eph));
-	rc = rdb_tx_update(&tx, &cont->c_prop, &ds_cont_prop_ec_agg_eph, &value);
-	if (rc == 0)
-		rc = rdb_tx_commit(&tx);
-	DL_CDEBUG(rc != 0, DLOG_ERR, DB_MD, rc, DF_CONT ": rdb_tx_update ec_agg_eph " DF_X64,
-		  DP_CONT(svc->cs_pool_uuid, cont_uuid), ec_agg_eph);
+	d_iov_set(&value, &old_eph, sizeof(old_eph));
+	rc = rdb_tx_lookup(&tx, &cont->c_prop, &ds_cont_prop_ec_agg_eph, &value);
+	if (rc == -DER_NONEXIST) {
+		rc = 0;
+		old_eph = 0;
+	}
+	if (rc != 0)
+		goto out;
 
-	cont_put(cont);
+	if (ec_agg_eph > old_eph) {
+		d_iov_set(&value, &ec_agg_eph, sizeof(ec_agg_eph));
+		rc = rdb_tx_update(&tx, &cont->c_prop, &ds_cont_prop_ec_agg_eph, &value);
+		if (rc == 0)
+			rc = rdb_tx_commit(&tx);
+	} else {
+		D_DEBUG(DB_MD, DF_CONT ": bypass rdb update ec_agg_eph "DF_X64", in rdb eph "DF_X64,
+			DP_CONT(svc->cs_pool_uuid, cont_uuid), ec_agg_eph, old_eph);
```

Comment:
[Question] In this case, `rc` is 0, and `cont_agg_eph_sync` will set `ea_rdb_eph` to the value of `ec_agg_eph` here, which <= `old_eph`, and then use the lower of the two values for refreshing. Is that okay?

Reply:
I'll check and refine it later after do some test for another issue, thx